server/implementation/src/main/java/io/smallrye/graphql/bootstrap/Bootstrap.java

```diff
@@ -113,6 +118,35 @@ private Bootstrap(Schema schema, Config config) {
         this.dataFetcherFactory = new DataFetcherFactory(config);
         this.eventEmitter = EventEmitter.getInstance(config);
         SmallRyeContext.setSchema(schema);
+        if (!Boolean.getBoolean("test.skip.injection.validation")) {
```

Comment:
Not completely sure about this solution, we could add a new property to the `Config`, but given that it's meant to be internal and testing-only, it sounds bad.

Reply:
This seems ok for now.